Modules/vc-module-notifications/VirtoCommerce.NotificationsModule.LiquidRenderer/LiquidTemplateRenderer.cs

```diff
@@ -25,7 +25,7 @@ public async Task<string> RenderAsync(string stringTemplate, object model, strin
             };
             var scriptObject = AbstractTypeFactory<NotificationScriptObject>.TryCreateInstance();
             scriptObject.Import(model);
-            scriptObject.Language = language;
+            scriptObject.Add(nameof(NotificationScriptObject.Language), language);
```

Comment:
This line also seems  redundant, `scriptObject` has a logic that allows to access it type properties without this constructions.

Reply:
https://github.com/lunet-io/scriban/blob/5718821436a31c0e212452921fbee3fb045aebca/src/Scriban/Runtime/ScriptObject.cs#L21 no, it's need to